Gemfile

```diff
@@ -0,0 +1,4 @@
+source "https://rubygems.org"
+
+# Specify your gem dependencies in inspec_tools.gemspec
+gemspec
```

Comment:
I generally try to stay consistent with newlines at the end of every file. It's normally a setting on your editor.